dbt/include/spark/macros/materializations/snapshot.sql

```diff
@@ -126,7 +126,7 @@
   {{ run_hooks(pre_hooks, inside_transaction=True) }}
 
   {% set strategy_macro = strategy_dispatch(strategy_name) %}
-  {% set strategy = strategy_macro(model, "snapshotted_data", "source_data", config, target_relation_exists) %}
+  {% set strategy = strategy_macro(model, "snapshotted_data", "source_data", model['config'], target_relation_exists) %}
```

Comment:
I don't think these are the same config. There's `config` in the globals, and then there's also `config.model.config`. Was there a bug or an issue that required this change?

Reply:
Yes.  I'm not sure why somebody was using the passed in config to start with (I suspect it was because using defaults is different for the config object and the config dictionary that got passed in), but you can't get a "property" from the dictionary, only the config object that you get from the context.